Do not disable any logger

cemsbr opened the pull request:
Fix #386

List of loggers that *were* disabled:
- concurrent.futures logger
- kytos.core.buffers logger
- kytos.core.napps.manager logger
- kytos.core.switch logger
- kytos.core.tcp_server logger
- socketio logger

cemsbr committed:
Do not disable any logger

Fix #386

List of loggers that *were* disabled:
- concurrent.futures logger
- kytos.core.buffers logger
- kytos.core.napps.manager logger
- kytos.core.switch logger
- kytos.core.tcp_server logger
- socketio logger

beraldoleal commented on kytos/core/logs.py:
```diff
@@ -23,7 +23,7 @@ def load_logging_file(cls, logging_file):
+            config.fileConfig(logging_file, disable_existing_loggers=False)
         except FileNotFoundError:
             cls.configuration.set('handler_syslog', 'args', '[]')
             config.fileConfig(cls.configuration)
```
Should not the line be here too?

cemsbr replied:
good catch. I refactored not to repeat code.

cemsbr committed:
[minor] LogManager: code review, bug fix, tests

- Review of commit 102481 ("Do not disable any logger", PR #430 to fix #386)
  - Add argument also to another fileConfig call.
- LogManager refactoring
  - Add tests
  - Not raising exception if there's no logging.ini file

Signed-off-by: Diego Rabatone Oliveira <diraol@ncc.unesp.br>

macartur commented on kytos/core/logs.py:
```diff
@@ -3,34 +3,32 @@
-        try:
-            config.fileConfig(logging_file)
-        except FileNotFoundError:
-            cls.configuration.set('handler_syslog', 'args', '[]')
```
I guess you forgot to verify the handler_syslog. This line change the 'handler_syslog' if the file /dev/syslog don't exists.

cemsbr committed:
[minor] LogManager review, refactoring, 100% ccov

Some details:
- Preventing errors when no syslog is available
- LogManager refactoring
- Added many comments and tests (100% coverage of logs module)
- Unified 2 files that was testing logs module

Signed-off-by: Diego Rabatone Oliveira <diraol@ncc.unesp.br>

cemsbr commented:
@beraldoleal and @macartur, I believe all the issues are solved now. There's also tests for this issues and 100% coverage of logs module.
